client/reader/tag-stream/controller.js

```diff
@@ -16,16 +16,31 @@ const analyticsPageTitle = 'Reader';
 
 export default {
 	tagListing( context ) {
-		var TagStream = require( 'reader/tag-stream/main' ),
-			basePath = '/tag/:slug',
-			fullAnalyticsPageTitle = analyticsPageTitle + ' > Tag > ' + context.params.tag,
-			tagSlug = trim( context.params.tag )
-				.toLowerCase()
-				.replace( /\s+/g, '-' )
-				.replace( /-{2,}/g, '-' ),
-			encodedTag = encodeURIComponent( tagSlug ).toLowerCase(),
-			tagStore = feedStreamFactory( 'tag:' + tagSlug ),
-			mcKey = 'topic';
+		var config = require( 'config' );
+
+		if ( config.isEnabled( 'reader/topics' ) ) {
```

Comment:
I was thinking we'd end up with a different stream object as part of this project rather than mucking with the current stream object and that would allow us to A/B test the two against each other. Is that not the case?

(though doing the if/else is still not necessary for this particular change, so maybe shouldn't do it anyways)

/tag does still work with this enabled.